tests/python/common/sampling/test_sampling.py

```diff
@@ -681,6 +681,117 @@ def _test3(p, replace):
         assert subg["flips"].num_edges() == 4
 
 
+def _test_sample_labors(hypersparse, prob):
```

Comment:
Is labor sampling GPU only or for both GPU and CPU?

Reply:
It supports CPU sampling, GPU UVA sampling, and pure UVA sampling, basically, everything NeighborSampler does. This PR mostly improves the performance of the GPU UVA sampling scenario.

Reply:
My intention was to make the change from NeighborSampler to LaborSampler as seamless as possible, they even have the same hyperparameters.